libs/damap/src/assets/i18n/dashboard/de.json

```diff
@@ -1 +1,73 @@
-{}
+{
+  "dashboard": {
+    "welcome.message": "Willkommen beim DMP Tool, einem Dienst, der Ihnen hilft, den Datenmanagementplan (DMP) für Ihr Projekt zu erstellen.",
+    "dmp": {
+      "heading": "Was ist ein Datenmanagementplan (DMP)?",
+      "text": {
+        "prefix": "Ein Datenmanagementplan (DMP) übersetzt datenbezogene Aspekte von ",
+        "dmp": {
+          "text": "DMP",
+          "link": "https://www.tuwien.at/en/research/rti-support/research-data/rdm-infos-tips/dmp"
+        },
+        "gsp": {
+          "text": "Guter Wissenschaftlicher Praxis ",
```

Comment:
Trailing or leading white space should not be part of the text, if it is a link (same for other links as well). In the UI, the white space will be highlighted as link as well.